x-pack/plugins/monitoring/server/kibana_monitoring/init.js

```diff
@@ -16,13 +15,11 @@ import { getCollectorTypesCombiner } from './lib';
  * @param {Object} kbnServer manager of Kibana services - see `src/server/kbn_server` in Kibana core
  * @param {Object} server HapiJS server instance
  */
-export function initBulkUploader(kbnServer, server) {
+export function initBulkUploader(_kbnServer, server) {
```

Comment:
If you updated the arg list here to `(kbnServer)` then you could just use `kbnServer.config.get(...)` below.